WordPress/src/main/java/org/wordpress/android/util/WPMediaUtils.java

```diff
@@ -256,34 +263,40 @@ public static void launchChooserWithContext(
                 requestCode);
     }
 
+    private static Intent preparePictureLibraryIntent(Context context, boolean multiSelect) {
+        return prepareIntent(context, multiSelect, Intent.ACTION_GET_CONTENT, "image/*",
+                new MimeTypes().getImageTypesOnly(), R.string.pick_photo);
+    }
+
     private static Intent prepareVideoLibraryIntent(Context context, boolean multiSelect) {
-        Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
-        intent.setType("video/*");
-        intent.putExtra(Intent.EXTRA_MIME_TYPES, new MimeTypes().getVideoTypesOnly());
-        if (multiSelect) {
-            intent.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true);
```

Comment:
This line in these intent methods was not getting the `multiSelect` param from the function so I modified it in the refactor. I checked and the behavior is the same. Let me know what you think as well.